probeinterface/io.py

```diff
@@ -83,7 +83,7 @@ def write_probeinterface(file, probe_or_probegroup):
     elif isinstance(probe_or_probegroup, ProbeGroup):
         probegroup = probe_or_probegroup
     else:
-        raise valueError('Bad boy')
+        raise ValueError('Bad boy')
```

Comment:
No comments on the Error message @JuliaSprenger? ehehehe

Reply:
There's so many comments on could make about this message. I couldn't decide for one :D

Reply:
I want to hear then all.